### PR TITLE
Add none property to resources

### PR DESCRIPTION
**Pull request by Mav-Ivan**
https://github.com/ita-social-projects/SpaceToStudy-Client/assets/110425368/87dd4805-78e4-49a1-b267-3ff8eaa90ec7

### CODE REVIEW

File: src/containers/my-resources/add-resource-with-input/AddResourceWithInput.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const AddResourceWithInput: FC<AddResourceWithInputProps> = ({
   button,
   selectedItems,
   setItems,
-  hideCategoriesFilter
+  showNoneProperty
```
Comment (Tolik170) [inline src/containers/my-resources/add-resource-with-input/AddResourceWithInput.tsx:44]: remove this prop, you don`t need to pass it from each resources container

**Review by abalanovsky — APPROVED** [review]

**Review by TSlashDreamy — APPROVED** [review]

File: src/containers/my-resources/add-resource-with-input/AddResourceWithInput.tsx (hunk 1 above)
Comment (Mav-Ivan) [reply src/containers/my-resources/add-resource-with-input/AddResourceWithInput.tsx:44]: done

File: src/containers/my-quizzes/QuizzesContainer.tsx [hunk 2]
```diff
@@ -101,6 +101,7 @@ const QuizzesContainer = () => {
         searchRef={searchTitle}
         selectedItems={selectedItems}
         setItems={setSelectedItems}
+        // showNoneProperty
```
Comment (Tolik170) [inline src/containers/my-quizzes/QuizzesContainer.tsx:104]: remove

**Review by MariaAfonina — APPROVED** [review]

File: src/containers/my-quizzes/QuizzesContainer.tsx (hunk 2 above)
Comment (Mav-Ivan) [reply src/containers/my-quizzes/QuizzesContainer.tsx:104]: yeah, thanks

File: src/components/filter-selector/FilterSelector.tsx [hunk 3]
```diff
@@ -79,15 +81,22 @@ const FilterSelector = <T extends Pick<CategoryNameInterface, '_id'>>({
...
-  )
+  const filteredItems = useMemo(() => {
+    const filtred = response.filter((item) =>
+      String(valueField ? item[valueField] : item)
+        .toLowerCase()
+        .includes(inputValue.toLowerCase())
+    )
+    return showNoneProperty
+      ? [
+          {
```
Comment (maarekua) [inline src/components/filter-selector/FilterSelector.tsx:92]: Move default item to variable.

Comment (Mav-Ivan) [reply src/components/filter-selector/FilterSelector.tsx]: done

**Pull request merged**